### PR TITLE
Add http interfaces for flush and compaction

### PR DESCRIPTION
**Pull request by dangleptr**

### CODE REVIEW

**Comment by dangleptr** [comment]
Jenkins go

File: src/storage/StorageHttpAdminHandler.cpp [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        resp_ = "Space should not be empty. Usage: http:://ip:port/admin?space=xx&op=yy";
+        return;
+    }
+    auto* op = headers->getQueryParamPtr("op");
+    if (op == nullptr) {
+        err_ = HttpCode::SUCCEEDED;
+        resp_ = "Op should not be empty. Usage: http:://ip:port/admin?space=xx&op=yy";
+        return;
+    }
+    if (schemaMan_ == nullptr || kv_ == nullptr) {
```
Comment (wadeliuyi) [inline src/storage/StorageHttpAdminHandler.cpp]: maybe can move this check to front of this function

File: src/kvstore/NebulaStore.h [hunk 2]
```diff
@@ -151,7 +151,9 @@ class NebulaStore : public KVStore, public Handler {
                            const std::string& configKey,
                            const std::string& configValue);
 
-    ResultCode compactAll(GraphSpaceID spaceId);
+    ResultCode compactAll(GraphSpaceID spaceId) override;
```
Comment (whitewum) [inline src/kvstore/NebulaStore.h]: compactALL sounds like compact all spaces

Comment (dangleptr) [reply src/kvstore/NebulaStore.h]: Good point

File: src/storage/StorageHttpAdminHandler.cpp (hunk 1 above)
Comment (dangleptr) [reply src/storage/StorageHttpAdminHandler.cpp]: Good point

File: src/storage/StorageHttpAdminHandler.cpp [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        // Unsupported method
+        err_ = HttpCode::E_UNSUPPORTED_METHOD;
+        return;
+    }
+    if (schemaMan_ == nullptr || kv_ == nullptr) {
+        err_ = HttpCode::SUCCEEDED;
+        resp_ = folly::stringPrintf("Error inside");
+        return;
+    }
+    auto* space = headers->getQueryParamPtr("space");
```
Comment (darionyaphet) [inline src/storage/StorageHttpAdminHandler.cpp]: Could use `headers->hasQueryParam` to check whether the argument is setting ?

File: src/storage/StorageHttpAdminHandler.cpp [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    if (space == nullptr) {
+        err_ = HttpCode::SUCCEEDED;
+        resp_ = "Space should not be empty. Usage: http:://ip:port/admin?space=xx&op=yy";
+        return;
+    }
+    auto* op = headers->getQueryParamPtr("op");
+    if (op == nullptr) {
+        err_ = HttpCode::SUCCEEDED;
```
Comment (darionyaphet) [inline src/storage/StorageHttpAdminHandler.cpp]: When some argument is missing, why not `E_ILLEGAL_ARGUMENT` ?

**Comment by dangleptr** [comment]
Jenkins go

**Comment by jude-zhu** [comment]
subtask of #724

**Pull request merged**